File: eau00.py
def combination():
	nb_one = 0
	nb_two = nb_one + 1
	nb_three = nb_two + 1

	for first_nb in range(nb_one, 8):
		nb_two = nb_one + 1
		for second_nb in range(nb_two, 9):
			nb_three = nb_two + 1
			for thrird_nb in range(nb_three, 10):
				print_numbers(first_nb, second_nb, thrird_nb)
				nb_three += 1
				if nb_one != 7:
					print(", ",end="")
			nb_two += 1
		nb_one += 1

def print_numbers(one, two, three):
	print("%d%d%d" % (one, two, three), end="")


combination()

File: eau00.js
// fonctions utilisées

function combination()
{
	let tab_number = [];
	let nb_one = 0;
	let nb_two = nb_one + 1;
	let nb_three = nb_two + 1;

	while (nb_one < 8)
	{
		nb_two = nb_one + 1;
		while (nb_two < 9)
		{
			nb_three = nb_two + 1;
			while (nb_three < 10)
			{
				tab_number.push(print_numbers(nb_one, nb_two, nb_three));
				nb_three++;
			}
			nb_two++;
		}
		nb_one++;
	}
	console.log(tab_number.join(', '));
}

function print_numbers(one, two, three)
{
	const number = [one,two,three];
	const new_number = number.join('')
	return new_number;
}


// Partie 1 : Gestion d'erreur
// Partie 2 : Parsing
// Partie 3 : Résolution
combination();
// Partie 4 : Affichage